cmd/importer/README.md

```diff
@@ -166,9 +166,11 @@ MySQL [test]> select * from t;
 10 rows in set (0.002 sec)
 ```
 
+`probability` controls the exceptions of `incremental` and `repeats`.
```

Comment:
Explain the relationship between the amount of `exception` and `probability`? Seems that the more probability, the fewer exceptions.